### DISCUSSION TITLE
Ignoring the Cookie Modal Window

### DISCUSSION THREAD

**Discussion by korogod08** [gildas-lormeau/single-file-cli#137] [Q&A]
When saving the page via the command line, the page is saved with the Cookie Policy modal window open. Is there a way I can bypass this window? I’ve tried various options in the Help section, but none of the solutions worked. Thank you.

**Comment by gildas-lormeau** [comment]
There is no generic way to remove them. I've just implemented a new option `--blocked-url-pattern` which will allow you to block requests. This option will be available in the next version. It can help to block the script which creates the modal popup. Alternatively, you could create a separate user profile in Chrome, install an extension to remove cookie banners into this profile, and pass the path to the user profile on the disk with the options `--browser-arg --headless=new --browser-arg --user-data-dir=<path to the profile folder>`.
